### DISCUSSION TITLE
Fritzbox 7272 - wie flashen?

### DISCUSSION THREAD

**Discussion by BeJoNe** [Freetz-NG/freetz-ng#194] [Allgemein]
Hallo, ich versuche vergebens die 7272 zu flashen.
Ich habe erfolgreich (mit Hilfe der EVA-tools von PeterPawn) die Box in den ftp modus versetzt.
Danach versuche ich mit
`EVA-FTP-Client.ps1 -Verbose -Debug -ScriptBlock { SwitchSystem }` damit linux_fs_start=0 ist
danach
`EVA-FTP-Client.ps1 -Verbose -Debug -ScriptBlock { GetEnvironmentValue }`
allerdings finde ich keinen Hinweis auf welches device ich das image hochladen soll?
`EVA-FTP-Client.ps1 -Verbose -Debug -ScriptBlock { UploadFlashFile c:\temp\7272.kernel mtd0 }` ?
Wo finde ich diese Info über die 7272?
Danke
Bernhard

**Comment by fda77** [comment]
> Wo finde ich diese Info über die 7220?

Keine Ahnung, du bist er Erste der so eine hat! Kannst du ein paar Bilder posten?
Falls die eine der 5 bekannten Flashmethoden unterstützt sollte das Script 'tools/push_firmware' weiterhelfen
Die x2xx Reihe hatte nur 1 System und EVA-FTP-Client.ps1 passt nicht dazu

**Reply by BeJoNe** [reply]
Kommando zurück, es ist eine 7272..

**Reply by fda77** [reply]
das Script 'tools/push_firmware' sollte weiterhelfen
Die x2xx Reihe hatte nur 1 System und EVA-FTP-Client.ps1 passt nicht dazu

**Reply by Conan179** [reply]
> Die x2xx Reihe hatte nur 1 System und EVA-FTP-Client.ps1 passt nicht dazu

Meinst du damit, die firmware partitionen?

**Reply by fda77** [reply]
Die hat nur 1 Partition

**Reply by Conan179** [reply]
Ähm, dann scheint die anzeige meiner 7272 falsch zu sein 
![freetz-ng 7272b](https://user-images.githubusercontent.com/1906242/107990249-4ff2e380-6fd4-11eb-8925-e3fcc933af74.JPG)
![freetz-ng 7272a](https://user-images.githubusercontent.com/1906242/107990263-55502e00-6fd4-11eb-9d48-02f6ee21690e.JPG)

**Reply by fda77** [reply]
Ach, die hat schon das "nand" Layout? Das muss man aber nicht wissen, da das Scirpt automatisch erkennt was passt:
> Falls die eine der 5 bekannten Flashmethoden unterstützt sollte das Script 'tools/push_firmware' weiterhelfen

**Reply by Conan179** [reply]
Ich hab sie jetzt seid 7jahren., raus kam sie August 2013 fast 4 jahre nach der 7270...

**Reply by fda77** [reply]
"EVA-FTP-Client.ps1" passt somit ebenso und mag übrigens nur inmemory images: `tools/image2inmemory`
Schon komisch dass AVM dieser das Format wie der 7490 verpasst hat, aber die 7390 hat es noch nicht

**Reply by Conan179** [reply]
Ich glaube, die 7272 sollte die kleine schwester der 7490 darstellen, sie hatte ein anderes Gehäuse und nur 2,5ghz. Ich hatte die damals von 1und1 bekommen, nach zwei 7390, da gab es die 7490 noch nicht bei 1und1, glaube ich.

**Reply by NDiGH** [reply]
> Schon komisch dass AVM dieser das Format wie der 7490 verpasst hat, aber die 7390 hat es noch nicht

Ganz einfach, die 7272 ist neuer als die 7490. Nicht immer nur von der Modellnummer auf das Alter schließen, das würde z.B. bei der 7570 auch nach hinten losgehen. ;-)

**Reply by BeJoNe** [reply]
dh. wie gehe ich vor?
Ich nehme nur das tools/image2inmemory und welche paramter?
schalte ich zuerst auf linux_fs_start=0 ? und welche partition mtd??

**Reply by NDiGH** [reply]
> dh. wie gehe ich vor?

Letztlich so wie bei einer 7490. Wenn du dazu push_firmware nicht verwenden möchtest dann siehe auch hier:
https://web.archive.org/web/20190529205946/https://www.ip-forum.eu/howto-fritz-box-firmware-images-auch-unsignierte-ueber-den-bootloader-installieren-577

**Reply by fda77** [reply]
> Ich nehme nur das tools/image2inmemory und welche paramter?

Das Image das du umwandeln möchtest. Ohne Parameter ist es das letzte erstelle Image
